Project.toml

```
@@ -29,7 +29,7 @@ Graphs = "1"
 InfrastructureModels = "0.7.3, 0.7.5"
 Ipopt = "0.9, 1.0.2, 1.1"
 JSON = "0.18, 0.19, 0.20, 0.21"
-JuMP = "0.22, 0.23, 1"
+JuMP = ">=1.15,<2"
```

```suggestion
JuMP = "1.15"
```
This should be equivalent.